contracts/interfaces/IDCAStrategies.sol

```diff
@@ -6,10 +6,26 @@ import '@mean-finance/dca-v2-core/contracts/interfaces/IDCAHub.sol';
 import '@mean-finance/nft-descriptors/solidity/interfaces/IDCAHubPositionDescriptor.sol';
 
 interface IDCAStrategiesManagementHandler {
+  /**
+   * @notice Emitted when a new strategy is created
+   * @param strategyId The id of the new strategy
+   * @param strategy The struct with the values of the new strategy
+   */
+  event StrategyCreated(uint80 strategyId, Strategy strategy, IDCAStrategies.ShareOfToken tokens);
+
+  /// @notice Thrown when a provided array is empty
+  error LengthZero();
+
+  /// @notice Thrown when a provided strategy name already exist
+  error NameAlreadyExists();
+
+  /// @notice Thrown when a provided array of token shares is misconfigured
+  error BadTokenShares();
+
   struct Strategy {
     address owner;
     string name;
-    IDCAStrategies.ShareOfToken[] tokens;
```

Comment:
Let's not delete this. The idea for this struct is to be used for off-chain purposes, so it makes sense to have everything in one place